jc/lib.py

```diff
@@ -224,6 +224,7 @@
     'vmstat-s',
     'w',
     'wc',
+    'wg',
```

Comment:
It might be a good idea to use wg-show to account for other possible future wg command parsers (e.g. showconf)

Reply:
@joehacksalot Done, thanks for the heads up 🫡